ucm2/sof-soundwire/dmic.conf

```diff
@@ -0,0 +1,46 @@
+SectionDevice."Mic1" {
+	Comment "Digital Microphone"
+
+	Value {
+		# FIXME: value is not 100 to avoid duplicate priority reported
+		# by ucm-validator		
+		CapturePriority 101
+		CapturePCM "hw:${CardId},3"
+		If.3mic {
```

Comment:
Use `Regex` condition here with `cfg-dmics:[34]` string here. It simplifies the config.

Reply:
ack, done.